### PR TITLE
[HxListLayout] Multiple DataTemplate sections + Tabs navigation #125

### PR DESCRIPTION
**Pull request by Harvey1214**
The test page for this issue was created using only the functionality HxTabPanel offers at this time.

### CODE REVIEW

**Comment by hakenr** [comment]
@crdo Can you continue on this PR?

**Comment by crdo** [comment]
@hakenr ready for review and follow up.

**Comment by Harvey1214** [comment]
@crdo I was working on this issue before you committed the changes and I think it would be sounder to extract the different parts of the component into methods returning RenderFragments, so that the same code could be reused for Regular and Card mode. Let me know what you think, thanks for considering.
Screenshot (356)(note: "else if" will be used)

**Review by hakenr — COMMENTED** [review]
Extraction of the RenderFragments for better reusability?

e.g. https://www.syncfusion.com/faq/blazor/templated-components/how-do-you-define-a-render-fragment-using-a-razor-template-in-blazor

**Comment by crdo** [comment]
> @crdo I was working on this issue before you committed the changes and I think it would be sounder to extract the different parts of the component into methods returning RenderFragments, so that the same code could be reused for Regular and Card mode. Let me know what you think, thanks for considering.
> Screenshot (356)(note: "else if" will be used)

@Harvey1214 Sure thing. Already told @hakenr that there is a repeated code and it has to be refactored. You can go ahead.

File: Havit.Blazor.Components.Web.Bootstrap/Navigation/HxTabPanelRenderMode.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+namespace Havit.Blazor.Components.Web.Bootstrap
+{
+	/// <summary>
+	/// Render variations for <see cref="HxTabPanel"/>.
+	/// </summary>
+	public enum RenderMode
+	{
+		Standard = 0,
```
[the remark this answers is not in the document]
Comment (crdo) [reply Havit.Blazor.Components.Web.Bootstrap/Navigation/HxTabPanelRenderMode.cs:8]: Tbh, `Standard` sounds better to me in this case but `Regular` for the other cases. If you want to unify, go ahead, I have no preference.

[the remark this answers is not in the document]
Comment (hakenr) [reply Havit.Blazor.Components.Web.Bootstrap/Navigation/HxTabPanelRenderMode.cs:8]: If there was no `Standard` up to now, I would go for `Regular`. For now, `Regular` sounds good for sizes and `Standard` for types/variations/kinds/etc.

**Comment by Harvey1214** [comment]
@crdo Do we want to handle the styling when NavVariant is set to Tabs? - set "padding-bottom: 0; border-bottom: 0;" on the card header, so that the tab navigation aligns to the content, or is it more suitable to let the users of the component adjust the CardSettings themselves (in that case, I suggest adding a demo to the docs).

**Comment by hakenr** [comment]
@Harvey1214 / @crdo Please add standalone demo for new render-mode to [HxTabPanel](https://havit.blazor.eu/components/HxTabPanel) documentation (without HxListLayout).

**Comment by crdo** [comment]
> @crdo Do we want to handle the styling when NavVariant is set to Tabs? - set "padding-bottom: 0; border-bottom: 0;" on the card header, so that the tab navigation aligns to the content, or is it more suitable to let the users of the component adjust the CardSettings themselves (in that case, I suggest adding a demo to the docs).

There is Bootstrap's class `card-header-tabs` for that.

**Comment by hakenr** [comment]
@crdo  @Harvey1214 Merged master into this PR (branch)
Waiting for confirmation of DONE... :-D

**Review by hakenr — CHANGES_REQUESTED** [review]

**Review by hakenr — APPROVED** [review]
lgtm

**Pull request merged**